attributes/dotnet_agent.rb

```diff
@@ -1,8 +1,7 @@
-default['appdynamics']['dotnet_agent']['version'] = 'latest'
-default['appdynamics']['dotnet_agent']['source'] = 'http://localhost/AppdynamicsInstallers/' #'https://packages.appdynamics.com/machine/%{version}/JavaAppAgent.zip'
-default['appdynamics']['dotnet_agent']['checksum'] = '7acb3756147a1d5a13c49b107a890ea56a8eb4099fd793e498e34b6f0b5962dc' #nil
+default['appdynamics']['dotnet_agent']['version'] = '4.0.7.0'
```

Comment:
AppDynamics shouldn't ship a cookbook that's pinned to a specific older version. We don't want to update the cookbook on each hot fix / release, and we don't want to default customers to a version that might be outdated.